android/assets/jsons/Tutorials.json

```diff
@@ -66,10 +66,26 @@
 
   {
     "name": "Roads and Railroads",
-    "steps": [
-      "Connecting your cities to the capital by roads\n  will generate gold via the trade route.\nNote that each road costs 1 gold Maintenance per turn, and each Railroad costs 2 gold,\n  so it may be more economical to wait until the cities grow!"
+    "civilopediaText": [
+        {"text":"Roads reduce the Movement Points required to move between two tiles connected by Roads.","link":"Tile Improvements/Road"},
+        {"text":"Railroads improve this reduction even more!","link":"Tile Improvements/Railroad"},
+        {"text":"This greatly improves the speed you can get Units around the map."},
+        {"text":"Until you research the technology that removes it, Roads do not cross Rivers"},
+        {"text":"Connecting your cities to the capital by Roads and/or Railroads will generate Gold via the Trade Route.","link":"Tutorials/Trade Route"},
+        {"text":"However, since each Road and Railroad have a Maintenance Cost, it may be more economical to wait until the cities grow and only place where needed."},
     ]
   },
+  {
+      "name": "Trade Route",
+      "civilopediaText": [
+          {"text":"Connecting your cities to the capital by Roads and/or Railroads will generate Gold via the Trade Route. You earn more Gold the more Population the City and your Capital has."},
```

Comment:
Could we add something like "Cities that are connected will show a small {roadSymbol} underneath them"?
That would help explain what the symbol means and how to tell if a city is connected.

Reply:
See line 84-85. I couldn't find the exact image in yellow and block border in the code. I think we might make it on the fly.